examples/BUILD.bazel

```diff
@@ -177,6 +177,36 @@ java_fuzz_target_test(
     ],
 )
 
+java_binary(
+    name = "CommonsTextFuzzerHooks",
+    srcs = ["src/main/java/com/example/CommonsTextFuzzerHooks.java"],
+    create_executable = False,
+    deploy_manifest_lines = ["Jazzer-Hook-Classes: com.example.CommonsTextFuzzerHooks"],
+    deps = [
+        "//agent/src/main/java/com/code_intelligence/jazzer/api:hooks",
+    ],
+)
+
+# WARNING: This fuzz target uses a vulnerable version of Apache Commons Text, which could result in the execution
+# of arbitrary code during fuzzing if executed with an older JDK. Use at your own risk.
+java_fuzz_target_test(
+    name = "CommonsTextFuzzer",
+    srcs = [
+        "src/main/java/com/example/CommonsTextFuzzer.java",
+    ],
+    fuzzer_args = [
+        "-fork=8",
+        "-use_value_profile=1",
```

Comment:
Did you happen to check whether these arguments, especially `-use_value_profile=1`, actually shorten the time until the bug is found on average?

Reply:
Good point, will do some measurements. (I assumed that given the library expects some very specific textual input for it to run into the vulnerable path, it might be better to use value profiling but I never checked how it does without it - maybe the instrumentation overhead makes it not worthwhile, I lack the XP to tell, so yeah, happy to make some measurements :) )

Reply:
I made some measurements based on which my assumption was right. I observed that with value profiling on, the overall convergence is better, even though execution speed gradually decreases. Without value profile, exec / sec is sort of consistently high, but I expect the fuzzer explores a lot more uninteresting branches and inputs and overall delivers slower progress.

In my environment, the Commons Text example averaged about 50 minutes but with large deviation, the quickest run I have seen was 15 mins. Meanwhile, without value profiling, the average runtime was about 2 hours and even the quickest I have seen was 75 mins. I would keep the value profiling on.